internal/provider/document_index/data.go

```diff
@@ -2,13 +2,13 @@ package document_index
 
 import (
 	"context"
-	"fmt"
-	"net/http"
 
 	"github.com/hashicorp/terraform-plugin-framework/datasource"
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 	"github.com/hashicorp/terraform-plugin-log/tflog"
+
+	vellumclient "terraform-provider-vellum/internal/sdk/client"
```

Comment:
I inlined just the parts of the sdk (straight copy and paste from `vellum-client-go`) that we need for this provider. We could remove the dependency on the next PR